modules/core/src/main/java/io/fluo/core/impl/TxStats.java

```diff
@@ -135,20 +135,22 @@ void setFinishTime(long t) {
   public void report(MetricNames names, String status, Class<?> execClass, MetricRegistry registry) {
     String sn = execClass.getSimpleName();
     if (getLockWaitTime() > 0) {
-      registry.timer(names.getTxLockwait() + sn).update(getLockWaitTime(), TimeUnit.MILLISECONDS);
+      registry.timer(names.getTxLockWaitTime() + sn).update(getLockWaitTime(),
+          TimeUnit.MILLISECONDS);
     }
-    registry.timer(names.getTxTime() + sn).update(getTime(), TimeUnit.MILLISECONDS);
+    registry.timer(names.getTxExecTime() + sn).update(getTime(), TimeUnit.MILLISECONDS);
     if (getCollisions() > 0) {
-      registry.histogram(names.getTxCollisions() + sn).update(getCollisions());
+      registry.meter(names.getTxWithCollision() + sn).mark();
```

Comment:
Marking zero made sense when it was a histogram and you want to find the average number of collisions. Now that the metrics are meters, I think we should only mark when a collisions occurs.